components/motor/ulnstepper/28byj-48.go

```diff
@@ -0,0 +1,406 @@
+// Package stepper28byj48 implements a GPIO based stepper motor with uln2003 controler.
```

Comment:
Let's add some documentation to the top. things that should be included
0. Description of the motor. maybe even state how many steps/rotation a typical model will have and that there are variations with different gearing.
1. the whole ULN vs. 23BYJ. that this driver is meant for a 23BYJ-48 class of steppers powered by the ULN2003 driver.
2. Link to a spec sheet
3. We chose the 8-step "half step" transition table...so each step is 5.625/64 deg i believe.
4. any constants or other limitations like min sleep time between steps